### ISSUE TITLE
Results do not update while typing 10.15.4+

### ISSUE THREAD

**Issue by li-yuening** [w0lfschild/Flashlight#68]
**OS:** 10.15.4
**Verision:**
MacForge: 0.14.1(2826)
Flashlight: 1.2.0

<img width="792" alt="截屏2020-04-07 下午4 21 52" src="https://user-images.githubusercontent.com/12383502/78646618-e605aa00-78eb-11ea-92d6-417771bb3b62.png">
The param `20 words` cannot be passed to the plugin immediately.
The plugin works fine in debug tool.
<img width="838" alt="截屏2020-04-07 下午4 25 10" src="https://user-images.githubusercontent.com/12383502/78646913-5c0a1100-78ec-11ea-8f22-135d634ccbca.png">
Same things happened to all my custom plugins.

These plugins worked in 10.14.

**Comment by tim-kim** [comment]
+1
I'm seeing the same issue on 10.15.4 OS version. GoogleTranslate plugin will not refresh the webpage to actually show the translation. Either shows no results or only captures the first letter:

<img width="666" alt="Screen Shot 2020-04-25 at 10 34 48 PM" src="https://user-images.githubusercontent.com/18359197/80298933-9fb2a680-8745-11ea-824b-808043ae25df.png">

**Comment by w0lfschild** [comment]
Yeah, not sure if/when a fix will come but I'm definitely aware of this.
